pkg/querier/http.go

```diff
@@ -98,9 +97,17 @@ func (q *QuerierAPI) RangeQueryHandler(w http.ResponseWriter, r *http.Request) {
 		serverutil.WriteError(err, w)
 		return
 	}
-	if err := marshal.WriteQueryResponseJSON(result, w); err != nil {
+
+	if r.Header.Get("Accept") == queryrange.ProtobufType {
+		// TODO: should this rather be: application/vnd.google.protobuf; proto=queryrange.QueryResponse
```

Comment:
I vote to leave it as `application/vnd.google.protobuf`

Reply:
Agreed; `Accept` headers should be for content types